vertical-pod-autoscaler/pkg/admission-controller/main.go

```diff
@@ -68,6 +70,7 @@ var (
 	registerWebhook    = flag.Bool("register-webhook", true, "If set to true, admission webhook object will be created on start up to register with the API server.")
 	registerByURL      = flag.Bool("register-by-url", false, "If set to true, admission webhook will be registered by URL (webhookAddress:webhookPort) instead of by service name")
 	vpaObjectNamespace = flag.String("vpa-object-namespace", apiv1.NamespaceAll, "Namespace to search for VPA objects. Empty means all namespaces will be used.")
+	allowedResources   = flag.String("allowed-resources", strings.Join([]string{string(apiv1.ResourceCPU), string(apiv1.ResourceMemory)}, ","), "Comma-separated list of resources that can be applied from a VPA.")
```

Comment:
We could have avoided that part of the patch because we are not making the Network recommendation anymore...
But if we can carry it over maybe that will help if we reopen for Network or Storage recommendations.